File: pygments/pygments_base16/base24-molokai.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#121212'
    base01 = '#121212'
    base02 = '#555555'
    base03 = '#6e6e6e'
    base04 = '#888888'
    base05 = '#a1a1a1'
    base06 = '#bbbbbb'
    base07 = '#ffffff'
    base08 = '#fa2573'
    base09 = '#dfd460'
    base0a = '#00afff'
    base0b = '#97e123'
    base0c = '#42a7cf'
    base0d = '#0f7fcf'
    base0e = '#8700ff'
    base0f = '#7d1239'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('molokai', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords

File: crosh/colors/base24-molokai.js
// Base16 Molokai
// Scheme: Iterm2B24

var color_scheme = {
        'base00': '#121212',
        'base01': '#121212',
        'base02': '#555555',
        'base03': '#6e6e6e',
        'base04': '#888888',
        'base05': '#a1a1a1',
        'base06': '#bbbbbb',
        'base07': '#ffffff',
        'base08': '#fa2573',
        'base09': '#dfd460',
        'base0A': '#00afff',
        'base0B': '#97e123',
        'base0C': '#42a7cf',
        'base0D': '#0f7fcf',
        'base0E': '#8700ff',
        'base0F': '#7d1239',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(161, 161, 161, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);
